scss/bitstyles/atoms/badge/Badge.stories.js

```diff
@@ -269,3 +269,20 @@ PositiveButtonSmall.args = {
   onClick: dummyOnClick,
   sizeVariant: ['small'],
 };
+
+const TemplateBadgeWithLabel = (args) => {
+  const fragment = document.createElement('div');
```

Comment:
Maybe `wrapper` would be a better name, since this isn’t a DocumentFragment any more? Not a big deal though